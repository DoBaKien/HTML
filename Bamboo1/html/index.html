<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="../css/index.css">
</head>

<body>
    <div class="header">
        <div id="sidebar" class="sidebar">
            <div class="menu-header">
                <img src="../assets/img/logo-color.png" alt="">
                <div class="box-icon">
                    <div class="lang">
                        <p>EN</p> | <p>VN</p>
                    </div>
                    <i class="icon-f ri-search-line"></i>
                    <button class="closebtn" onclick="closeNav()">
                        <i class="ri-close-line"></i>
                    </button>
                </div>
            </div>
            <div class="list-menu">
                <div class="menu-item">
                    <a href="#">Về chúng tôi</a>
                    <div class="sub-menu">
                        <a href="#">Service 1</a>
                        <a href="#">Service 2</a>
                        <a href="#">Service 3</a>
                    </div>
                </div>
                <div class="menu-item">
                    <a href="#">Câu truyện của tre</a>
                    <div class="sub-menu">
                        <a href="#">Client 1</a>
                        <a href="#">Client 2</a>
                        <a href="#">Client 3</a>
                    </div>
                </div>
                <div class="menu-item">
                    <a href="#">Sản phẩm & dịch vụ</a>
                    <div class="sub-menu">
                        <a href="#">Email</a>
                        <a href="#">Phone</a>
                        <a href="#">Address</a>
                    </div>
                </div>
                <div class="menu-item">
                    <a href="#">Dự án</a>
                    <div class="sub-menu">
                        <a href="#">Email</a>
                        <a href="#">Phone</a>
                        <a href="#">Address</a>
                    </div>
                </div>
                <div class="menu-item">
                    <a href="#">Liên hệ</a>
                    <div class="sub-menu">
                        <a href="#">Email</a>
                        <a href="#">Phone</a>
                        <a href="#">Address</a>
                    </div>
                </div>
            </div>

        </div>
        <div id="main">
            <img src="../assets/img/logo.png" alt="" width="40px" height="40px">
            <button class="openbtn" onclick="openNav()"><i class="ri-menu-line"></i></button>
        </div>
    </div>
    <div class="asd"></div>
    <div id="footer">
        <div class="row1">
            <div class="item1">
                <img src="../assets/img/logo-color.png" alt="" width="130px">
                <div class="box-title">
                    <div class="title">BAMBOO</div>
                    <div class="subtitle">N U R S E R Y</div>
                </div>
            </div>
            <div class="item2">
                <div class="title">Dang ky cap nhat</div>
                <input placeholder="nhap email cua bn" />
            </div>
        </div>
        <div class="row2">
            <div class="col">
                <div class="title">Liên hệ</div>
                <p>
                    <i class="ri-map-pin-2-line"></i>
                    2C15, Đường Trường Sa, Phường 19, Quận Bình Thạnh, TP.HCM, Việt Nam
                </p>
                <p> <i class="ri-map-pin-2-line"></i> 1234556789</p>
                <p> <i class="ri-map-pin-2-line"></i> contact@bamboonursery.vn</p>
            </div>
            <div class="col">
                <div class="title">Liên hệ</div>
                <p>Giá trị cốt lõi</p>
                <p>Tầm nhìn chiến lược</p>
                <p>Trách nhiệm cộng đồng</p>
                <p>Đối tác đồng hành </p>
            </div>
            <div class="col">
                <div class="title">Câu chuyện của Tre</div>
                <p>Tại sao là Tre</p>
                <p>Chăm sóc Tre</p>
                <p>Kiểm soát Tre</p>
            </div>
            <div class="col">
                <div class="title">Sản phẩm & Dịch vụ</div>
                <p>Cây tre giống</p>
                <p>Sản phẩm khác</p>
                <p>Giải pháp toàn diện</p>
            </div>
            <div class="col">
                <div class="title">Chính sách & quyền lợi khách hàng</div>
                <p>Dự án của chúng tôi</p>
                <p>Bamboo Nursery’s map</p>
            </div>
        </div>
        <div class="row3">
            <div class="text">Ban quyen thuoc BambooNursery</div>
            <div class="box">
                <i class="icon-fb ri-facebook-fill"></i>
                <i class="icon-fb ri-facebook-fill"></i>
                <i class="icon-fb ri-facebook-fill"></i>
                <i class="icon-fb ri-facebook-fill"></i>
            </div>
        </div>
    </div>
    <script src="../js/index.js"></script>
</body>

</html>