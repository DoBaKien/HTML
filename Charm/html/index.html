<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" href="../css/style.css">
</head>

<body>
    <div class="container-fuild">
        <div class="row">
            <div id="tab-scroll" class="d-none d-md-flex">
                <ul id="simple-list-example" class="d-flex flex-column simple-list-example-scrollspy ">
                    <li>
                        <a class="rounded" href="#simple-list-item-1">1 </a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-2">2</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-3">3</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-4">4</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-5">5</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-6">6</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-7">7</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li><a class="rounded" href="#simple-list-item-8">8</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>

                </ul>
            </div>
            <div class="logo">
                <img src="../assets/imgs/icon-normal.svg" alt="">
            </div>
            <div class="col-12">
                <div data-bs-spy="scroll" data-bs-target="#simple-list-example" data-bs-offset="0"
                    data-bs-smooth-scroll="true" class="scrollspy-example" tabindex="0">
                    <div class="box-client-1 client" id="simple-list-item-1">
                        <div id="carouselExampleFade" class="carousel slide carousel-fade" data-bs-ride="carousel">
                            <div class="carousel-inner">
                                <div class="carousel-item active">
                                    <img src="../assets/imgs/hinh-nen-may-tinh-dep-full-hd-11.jpg" class="d-block w-100"
                                        alt="...">
                                    <div class="carousel-caption" id="item1">
                                        <h5>Charm Resort Long Hai</h5>
                                        <p>Đẳng cấp nghỉ dưỡng 5 sao</p>
                                        <div class="asd">Giữa lòng long hải</div>
                                    </div>
                                </div>
                                <div class="carousel-item">
                                    <img src="../assets/imgs/hinh-nen-may-tinh-dep-full-hd-13.jpg" class="d-block w-100"
                                        alt="...">
                                    <div class="carousel-caption" id="item2">
                                        <h5>Charm Resort Long Hai</h5>
                                        <p>Đẳng cấp nghỉ dưỡng 5 sao</p>
                                        <div class="asd">Giữa lòng long hải</div>
                                    </div>
                                </div>
                                <div class="carousel-item">
                                    <img src="../assets/imgs/hinh-nen-may-tinh-dep-full-hd-14.jpg" class="d-block w-100"
                                        alt="...">
                                    <div class="carousel-caption" id="item3">
                                        <h5>Charm Resort Long Hai</h5>
                                        <p>Đẳng cấp nghỉ dưỡng 5 sao</p>
                                        <div class="asd">Giữa lòng long hải</div>
                                    </div>
                                </div>
                            </div>


                        </div>

                    </div>
                    <div class="box-client-2 client" id="simple-list-item-2">
                        <div class="carousel" data-flickity='{ "fade": true, "imagesLoaded": true }'>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/720/540/?image=517" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/540/720/?image=696" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/720/540/?image=56" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/800/500/?image=1084" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/720/540/?image=1080" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/640/640/?image=1074" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/720/540/?image=1069" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/800/500/?image=1062" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/720/540/?image=1002" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/640/640/?image=935" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/720/540/?image=855" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image" src="https://picsum.photos/640/640/?image=824" />
                            </div>
                        </div>
                    </div>
                    <div class="box-client-3 client">
                        <h1 id="simple-list-item-3">3</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-4 client">
                        <h1 id="simple-list-item-4">4</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-5 client">
                        <h1 id="simple-list-item-5">5</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-6 client">
                        <h1 id="simple-list-item-6">6</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-7 client">
                        <h1 id="simple-list-item-7">7</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-8 client">
                        <h1 id="simple-list-item-8">8</h1>
                        <p>...</p>
                    </div>
                </div>
            </div>
            <div class="col-1" id="tab-left">
                <div class="box-line">
                    <div class="line"></div>
                    <div class="box-menu">
                        <div class="box-content">MENU</div>
                    </div>
                    <div class="carrer">TUYỂN DỤNG</div>
                </div>
                <div class="box-mid">
                    <p>English</p>
                    <div class="line"></div>
                    <p>Search</p>
                </div>
                <div class="box-bot">
                    <p id="scrollbtn">Scroll down</p>
                </div>
            </div>
        </div>
    </div>
    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>

    <script src="../js/script.js"></script>
</body>

</html>