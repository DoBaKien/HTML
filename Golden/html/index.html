<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="https://npmcdn.com/flickity@2/dist/flickity.css">
    <link rel="stylesheet" href="../css/style.css">
</head>

<body>

    <div class="box1 reveal" id="box1">
        <header class="container" id="header">
            <div class="box-mobi">
                <img src="../assets/img/icon-normal.svg" alt="" class="logo-mobi">
                <p class="name"> GOLDEN SUN</p>
            </div>
            <div class="box-icon">
                <div class="contact">
                    <div class="box">
                        <i class="phone-icon ri-phone-fill"></i>
                        <p class="phone"> 0123456789</p>
                        <i class="icon-h ri-home-8-fill"></i>
                    </div>
                </div>
                <img src="../assets/img/icon-normal.svg" alt="" class="logo" id="logo-header">
                <div class="menu">
                    <div class="box-menu" id="box-menu">
                        <p id="men">Menu</p>
                        <div class="hamburger">
                            <span class="span-1" id="span-1"></span>
                            <div class="span-2" id="box-span">
                                <span class="span-21"></span>
                                <span class="span-22"></span>
                            </div>
                            <span class="span-1" id="span-11"></span>
                        </div>
                    </div>
                    <div class="box-menu-1">En</div>
                </div>

            </div>
        </header>
        <nav class="menu-container" id="menu">
            <div class="image-box">
                <img src="../assets/img/icon-normal.svg" alt="" id="rotateImage">
            </div>
            <ul>
                <li class="li1"><a href="" data-text="TRANG CHỦ">TRANG CHỦ</a></li>
                <li class="li1">
                    <a href="" data-text="VỀ CHÚNG TÔI">VỀ CHÚNG TÔI</a>
                </li>
                <li class="li1">
                    <a href="" data-text="LĨNH VỰC HOẠT ĐỘNG">LĨNH VỰC HOẠT ĐỘNG</a>

                </li>
                <li class="li1"><a href="" data-text="DỰ ÁN ĐẦU TƯ">DỰ ÁN ĐẦU TƯ</a></li>
                <li class="li2"><a href="" data-text="PHÁT TRIỂN NHÂN LỰC">PHÁT TRIỂN NHÂN LỰC</a></li>
                <li class="li2"><a href="" data-text="TRUYỀN THÔNG">TRUYỀN THÔNG</a></li>
                <li class="li2"><a href="" data-text="LIÊN HỆ">LIÊN HỆ</a></li>
            </ul>
        </nav>
        <div class="box-logo" id="box-logo">
            <img src="../assets/img/icon-normal.svg" alt="">
            <div class="title1">GOLDEN SUN</div>
        </div>
        <div class="background"></div>
        <div class="title" id="title">GOLDEN SUN</div>
        <div class="background-bottom"></div>
        <div class="tag">GOLDEN SUN GOLDEN SUN GOLDEN SUN</div>
    </div>
    <div class="box2 reveal" id="box2">
        <div class="text1">
            <span class="char" id="char-1">G</span>
            <span class="char" id="char-2">H</span>
            <span class="char" id="char-3">I</span>
            &#8200&#8200
            <span class="char" id="char-4">D</span>
            <span class="char" id="char-5">Ấ</span>
            <span class="char" id="char-6">U</span>
            &#8200&#8200
            <span class="char" id="char-7">Đ</span>
            <span class="char" id="char-8">Ậ</span>
            <span class="char" id="char-9">M</span>
            &#8200&#8200
            <span class="char" id="char-10">N</span>
            <span class="char" id="char-11">É</span>
            <span class="char" id="char-12">T</span>
            <br><br>
            <span class="char" id="char-13">T</span>
            <span class="char" id="char-14">R</span>
            <span class="char" id="char-15">Ê</span>
            <span class="char" id="char-16">N</span>
            &#8200&#8200
            <span class="char" id="char-17">B</span>
            <span class="char" id="char-18">Ả</span>
            <span class="char" id="char-19">N</span>
            &#8200&#8200
            <span class="char" id="char-20">Đ</span>
            <span class="char" id="char-21">Ồ</span>
            <br><br>

            <div class="box-char">
                <div class="bds">
                    <span class="char asd" id="char-22">B</span>
                    <span class="char asd" id="char-23">Ấ</span>
                    <span class="char asd" id="char-24">T</span>
                    &#8200&#8200
                    <span class="char asd" id="char-25">Đ</span>
                    <span class="char asd" id="char-26">Ộ</span>
                    <span class="char asd" id="char-27">N</span>
                    <span class="char asd" id="char-28">G</span>
                    &#8200&#8200
                    <span class="char asd" id="char-29">S</span>
                    <span class="char asd" id="char-30">Ả</span>
                    <span class="char asd" id="char-31">N</span>
                </div>

                <div class="vn">
                    <span class="char" id="char-32">V</span>
                    <span class="char" id="char-33">I</span>
                    <span class="char" id="char-34">Ệ</span>
                    <span class="char" id="char-35">T</span>
                    &#8200&#8200
                    <span class="char" id="char-36">N</span>
                    <span class="char" id="char-37">A</span>
                    <span class="char" id="char-38">M</span>
                </div>
            </div>

        </div>
        <div class="text2">
            <div class="chart">
                <span class="char" id="chart-1">B</span>
                <span class="char" id="chart-2">Ằ</span>
                <span class="char" id="chart-3">N</span>
                <span class="char" id="chart-4">G</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-5">N</span>
                <span class="char" id="chart-6">H</span>
                <span class="char" id="chart-7">Ữ</span>
                <span class="char" id="chart-8">N</span>
                <span class="char" id="chart-9">G</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-10">D</span>
                <span class="char" id="chart-11">Ự</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-12">Á</span>
                <span class="char" id="chart-13">N</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-14">T</span>
                <span class="char" id="chart-15">Ầ</span>
                <span class="char" id="chart-16">M</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-17">C</span>
                <span class="char" id="chart-18">Ỡ</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-19">V</span>
                <span class="char" id="chart-20">À</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-21">T</span>
                <span class="char" id="chart-22">H</span>
                <span class="char" id="chart-23">Â</span>
                <span class="char" id="chart-24">N</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-25">T</span>
                <span class="char" id="chart-26">H</span>
                <span class="char" id="chart-27">I</span>
                <span class="char" id="chart-28">Ệ</span>
                <span class="char" id="chart-29">N</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-30">V</span>
                <span class="char" id="chart-31">Ớ</span>
                <span class="char" id="chart-32">I</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-33">M</span>
                <span class="char" id="chart-34">Ô</span>
                <span class="char" id="chart-35">I</span>
            </div>

            <div class="chart">
                <span class="char" id="chart-36">T</span>
                <span class="char" id="chart-37">R</span>
                <span class="char" id="chart-38">Ư</span>
                <span class="char" id="chart-39">Ờ</span>
                <span class="char" id="chart-40">N</span>
                <span class="char" id="chart-41">G</span>
            </div>
            <br><br>
        </div>
    </div>
    <div class="box3 reveal" id="box3">
        <div class="box-char">
            <div class="linhvuc">
                <span class="char asd" id="char-1">L</span>
                <span class="char asd" id="char-2">Ĩ</span>
                <span class="char asd" id="char-3">N</span>
                <span class="char asd" id="char-4">H</span>
                &#8200 &#8200
                <span class="char asd" id="char-5">V</span>
                <span class="char asd" id="char-6">Ự</span>
                <span class="char asd" id="char-7">C</span>
            </div>
            <div class="hoatdong">
                <span class="char asd" id="char-8">H</span>
                <span class="char asd" id="char-9">O</span>
                <span class="char asd" id="char-10">Ạ</span>
                <span class="char asd" id="char-11">T</span>
                &#8200 &#8200
                <span class="char asd" id="char-12">Đ</span>
                <span class="char asd" id="char-13">Ộ</span>
                <span class="char asd" id="char-14">N</span>
                <span class="char asd" id="char-15">G</span>
            </div>
        </div>
        <div class="box">
            <div class="box-content">
                <ul>
                    <li id="li-1">
                        <a href="">
                            <span>1/</span> nghĩ dưỡng
                        </a>
                    </li>
                    <li id="li-2">
                        <a href="">
                            <span>2/</span> bất động sản
                        </a>
                    </li>
                    <li id="li-3">
                        <a href="">
                            <span>3/</span> truyền hình viễn thông
                        </a>
                    </li>
                    <li id="li-4">
                        <a href="">
                            <span>4/</span> du lịch sinh thái
                        </a>
                    </li>
                    <li id="li-5">
                        <a href="">
                            <span>5/</span> khu vui chơi giải trí
                        </a>
                    </li>
                </ul>
            </div>

            <div class="box-img">
                <img src="../assets/img/icon-normal.svg" alt="" id="img-box3">
            </div>
        </div>

    </div>
    <div class="box4 reveal">
        <div class="box-content">
            <div class="box-title" id="box-text-1">
                <div class="content">
                    <p class="num" id="num-1">01</p>
                </div>
                <div class="box-left">
                    <p class="text" id="text-1">NGHỈ DƯỠNG</p>
                </div>
                <div class="btn">
                    <a href="">XEM THÊM</a>
                    <i class="ri-arrow-right-s-line"></i>
                </div>
            </div>

            <div class="box-image">
                <div class="img9"> <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt=""></div>
                <div class="box-action">
                    <div class="img-act">
                        <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt="">
                    </div>

                    <div class="act">
                        <div class="bg" id="bg">
                            <button id="timerButton" class="circle-button">
                                <span><i class="ri-arrow-left-line"></i></span>
                            </button>
                        </div>
                        <div class="bg" id="bg1">
                            <button id="timerButton1" class="circle-button">
                                <span><i class="ri-arrow-right-line"></i></span>
                            </button>
                        </div>
                    </div>
                </div>
            </div>

        </div>
        <div class="box-desc">

            <p>
                Bắt nhịp với dòng chảy thời cuộc và khát vọng tô điểm bản đồ bất động sản Việt Nam, BĐS
                Nghỉ Dưỡng là lĩnh vực hoạt động nòng cốt tại Golden Sun với nhiều nỗ lực. Golden Sun đã & đang kiến
                tạo
                nên những kiệt tác nghỉ dưỡng cao cấp, đẳng cấp quốc tế, cùng dịch vụ chuẩn mực mang đến trải nghiệm
                khác biệt, vượt kỳ vọng cho khách hàng.
            </p>

            <div class="btn">
                <a href="">XEM THÊM</a>
                <i class="ri-arrow-right-s-line"></i>
            </div>
        </div>
    </div>
    <div class="box5 reveal">
        <div class="box-title" id="box-text-2">

            <div class="box-left">
                <p class="text" id="text-2">BẤT <br> ĐỘNG <br> SẢN</p>
            </div>
            <div class="content">
                <p class="num" id="num-2">02</p>
            </div>
        </div>
        <div class="box-content">
            <div class="img-content">
                <img src="../assets/img/1.png" alt="">
            </div>
            <div class="text-content">
                <p>Bất động sản là lĩnh vực giúp đưa tên tuổi Golden Sun ghi dấu ấn trên thị trường. Phát huy tối đa nội
                    lực doanh nghiệp và tìm kiếm các đơn vị hợp tác uy tín, Golden Sun xây dựng nền tảng vững chắc với
                    hoạt động đầu tư và cho thuê Bất động sản.</p>
                <div class="btn">
                    <a href="">XEM THÊM</a>
                    <i class="ri-arrow-right-s-line"></i>
                </div>
            </div>
        </div>
    </div>
    <div class="box6 reveal">
        <div class="box-content">
            <div class="box-title" id="box-text-3">
                <div class="content">
                    <p class="num" id="num-3">03</p>
                </div>
            </div>
            <div class="bg"></div>
            <div class="title">
                <div class="text">
                </div>
                <div class="p-title">
                    <p>Truyền hình viễn thông</p>
                    <div class="title-img"><img src="../assets/img/1.jpg" alt=""></div>
                </div>
                <div class="text">
                    <div class="desc">
                        Golden Sun sở hữu Công ty Truyền hình Cáp Nghệ An thương hiệu quen thuộc và uy tín đối
                        với đông đảo khán giả tại Nghệ An. Cung cấp dịch vụ internet, lắp đặt và quản lý các kênh,
                        chương trình phát sóng
                    </div>
                    <div class="btn">
                        <a href="">XEM THÊM</a>
                        <i class="ri-arrow-right-s-line"></i>
                    </div>
                </div>
            </div>
            <div class="box-img"><img src="../assets/img/1.jpg" alt=""></div>
        </div>

    </div>
    <div class="box7 reveal">
        <div class="box-content">
            <div class="box-title" id="box-text-4">
                <div class="content">
                    <p class="num" id="num-4">04</p>
                </div>
                <div class="box-left">
                    <p class="text" id="text-4">DU LỊCH SINH THÁI</p>
                </div>
                <div class="btn">
                    <a href="">XEM THÊM</a>
                    <i class="ri-arrow-right-s-line"></i>
                </div>
            </div>
            <div class="box-image">
                <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt="">
                <div class="act">
                    <div class="bg" id="bg2">
                        <button id="timerButton2" class="circle-button">
                            <span><i class="ri-arrow-left-line"></i></span>
                        </button>
                    </div>
                    <div class="bg" id="bg3">
                        <button id="timerButton3" class="circle-button">
                            <span><i class="ri-arrow-right-line"></i></span>
                        </button>
                    </div>
                </div>
            </div>
        </div>
        <div class="decs" id="decs">
            <p class="numbi" id="numbi">04</p>
            <p class="desc">Trải nghiệm mới mẻ đắm mình trong vẻ đẹp của thiên nhiên</p>
        </div>
    </div>
    <div class="box8 reveal">
        <div class="box-content">
            <div class="box-title" id="box-text-5">
                <div class="content">
                    <p class="num" id="num-5">05</p>
                </div>
                <div class="box-left">
                    <p class="text" id="text-5">KHU <br> VUI CHƠI GIẢI TRÍ</p>
                </div>
            </div>
        </div>
        <div class="box-desc">
            <div class="img-desc">
                <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt="">
            </div>
            <div class="text-desc">
                <p>Đa dạng loại hình giải trí với nhiều tiện ích</p>
                <div class="btn">
                    <a href="">XEM THÊM</a>
                    <i class="ri-arrow-right-s-line"></i>
                </div>
            </div>
        </div>
    </div>
    <div class="box9 reveal" id="box9">
        <div class="box-container">
            <div class="box-img">
                <img src="../assets/img/icon-normal.svg" alt="" id="img-box9">
            </div>
            <div class="box-content">
                <div class="title">
                    <p>Dự án tiêu biểu</p>
                </div>
                <div class="desc">
                    <p>
                        VỚI NHỮNG CHUẨN MỰC LÀM NÊN THƯƠNG HIỆU: QUY CHUẨN, KINH NGHIỆM, TÂN TIẾN, CHÍNH TRỰC,
                        CHẤT LƯỢNG, GOLDEN SUN CAM KẾT MANG ĐẾN NHỮNG DỰ ÁN ĐẲNG CẤP QUỐC TẾ...
                    </p>
                </div>

            </div>
        </div>
    </div>
    <div class="box10 reveal">
        <div class="box-img">
            <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt="">
        </div>
        <div class="carousel" id="carousel">
            <!-- <div class="carousel-cell">
                <div class="box-container">
                    <div class="box-text">
                        <div class="text1">
                            <p> GOLDEN IMPERIAL <br> HOTEL MĂNG ĐEN</p>
                        </div>
                        <div class="text2">
                            <p>
                                Tọa lạc giữa “trái tim” thị trấn Măng Đen, huyện KonPlong tỉnh Kon Tum, Tựa
                                như bản giao hưởng tinh tế của đại ngàn Măng Đen, Golden Boutique Hotel là khách sạn
                                đẳng cấp nhất tại thị trấn Măng Đen, Kon Tum.
                            </p>
                            <div class="btn">
                                <a href="">XEM THÊM</a>
                                <i class="ri-arrow-right-s-line"></i>
                            </div>
                        </div>

                    </div>
                    <div class="img-cell">
                        <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt="">
                    </div>
                    <div class="box-title">
                        <div class="num">01</div>
                        <p>Một không gian sang trọng hiếm có với lối kiến trúc phong cách Tân Cổ điển độc bản</p>
                    </div>
                </div>
            </div> -->
        </div>

        <div class="act">
            <div class="bg" id="bg4">
                <button id="timerButton4" class="circle-button">
                    <span><i class="ri-arrow-left-line"></i></span>
                </button>
            </div>
            <div class="bg" id="bg5">
                <button id="timerButton5" class="circle-button">
                    <span><i class="ri-arrow-right-line"></i></span>
                </button>
            </div>
        </div>
    </div>

    <div class="box11 reveal">
        <div class="box-container">
            <div class="box-1">
                <p>TRUYỀN THÔNG</p>
            </div>
            <div class="box-2">
                <div class="box-21">
                    <div class="box-211">TRUYỀN <br> THÔNG</div>
                    <div class="box-212">
                        <p> CÁC THÔNG TIN MỚI NHẤT ĐƯỢC CẬP NHẬT THƯỜNG XUYÊN</p>
                    </div>
                </div>
                <div class="box-22" id="box-22">
                    <!-- <div class="box-content">
                        <div class="time">7-11-2024</div>
                        <div class="box">
                            <div class="img-box">
                                <img src="../assets/img/0106_hinh-nen-4k-may-tinh5.jpg" alt="">
                            </div>
                            <div class="content">
                                <p>Golden Imperial Hotel Đà Lạt: Bốn mùa hội tụ nơi thành phố ngàn hoa!</p>
                                <div class="btn">
                                    <a href="">Chi tiết</a>
                                    <i class="ri-arrow-right-s-line"></i>
                                </div>
                            </div>

                        </div>
                    </div> -->
                </div>
            </div>
        </div>
    </div>
    <div class="box12 reveal" id="box12">
        <div class="box-container">
            <div class="title">
                <p>ĐỐI TÁC</p>
            </div>

            <div class="list" id="list-icon">
                <!-- <div class="item">
                    <img src="../assets/img/icon-normal.svg" alt="">
                </div> -->
            </div>
        </div>
    </div>
    <div class="box13 reveal">
        <div class="marquee">
            <ul class="marquee__inner">
                <li class="marquee__part">
                    GOLDEN SUN
                </li>
                <li class="marquee__part">
                    GOLDEN SUN
                </li>
                <li class="marquee__part">
                    GOLDEN SUN
                </li>
                <li class="marquee__part">
                    GOLDEN SUN
                </li>
            </ul>
        </div>
        <div class="box-container">
            <div class="box-1">
                <div class="box-logo">
                    <img src="../assets/img/icon-normal.svg" alt="">
                    <div class="name">GOLDEN SUN</div>
                </div>
                <div class="name-page">LIÊN HỆ</div>
            </div>
            <div class="box-2">
                <ul>
                    <li>
                        <i class="ri-map-pin-line"></i>
                        <p>Số 25, Nguyễn Thị Minh Khai, Phường Lê Mao, Thành phố Vinh, Tỉnh Nghệ An, Việt Nam</p>
                    </li>
                    <li>
                        <i class="ri-phone-fill"></i>
                        <p>1234567890</p>
                    </li>
                    <li>
                        <i class="ri-mail-open-line"></i>
                        <p>123@gmail.com</p>
                    </li>
                </ul>
                <div class="box-info">
                    <form action="">
                        <input type="text" placeholder="HỌ VÀ TÊN">
                        <input type="number" placeholder="SỐ ĐIỆN THOẠI" id="phoneNumber">
                        <input type="text" placeholder="EMAIL">
                        <textarea id="w3review" rows="4"></textarea>
                        <div class="btn">
                            <button type="submits"> Gửi thông tin <i class="ri-arrow-right-s-line"></i></button>
                        </div>
                    </form>
                </div>
            </div>
        </div>
    </div>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
    <script src="https://npmcdn.com/flickity@2/dist/flickity.pkgd.js"></script>
    <script src="../js/script.js"></script>
</body>

</html>